services/userService.js

```diff
@@ -0,0 +1,129 @@
+const userDao = require('../models/userDao')
+const middleHash = require('../middleware/hash')
+const middleErr = require('../middleware/error')
+const middleJwt = require('../middleware/jwt')
+const middleMailer = require('../middleware/mailer')
+
+const signUp = async( userEmail, userPassword, userNickname, userPhoneNumber, userBirthDay, userGender ) => {
+    if(!userEmail.includes('@') || !userEmail.includes('.')){
+        middleErr.error(400, "EMAIL_FORMAT_INCORRECT")
+    }
+    if(userPassword.length<10){
+        middleErr.error(400, "PASSWORD_FORMAT_INCORRECT")
+    }
+    console.log()
+    if(userPhoneNumber.length<13){
+        middleErr.error(400, "PHONE_NUMBER_FORMAT_INCORRECT")
+    }
```

Comment:
```suggestion
    const maxPhoneNumberLength = 13;
    if(userPhoneNumber.length < maxPhoneNumberLength){
        middleErr.error(400, "PHONE_NUMBER_FORMAT_INCORRECT")
    }
```

13이라는 숫자가 어떤 의미를 가지고 있는지 개발자가 추측해야할 수도 있으므로, 그 의미를 명확히 할 수 있도록 별도의 변수로 분리하는 것이 좋겠습니다